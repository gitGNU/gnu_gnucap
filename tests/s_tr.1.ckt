'c check
v1 1 0 dc pulse(0 1 0 .01 .01 1k 1k) ac 1
r1 1 2 1
c2 2 0 1
.list
.print tran v(1) v(2)
.tran .1 1 0 trace all
.tran trace all
.tran .1 5 4
.tran .1 9 8 trace all
.tran trace all
.status notime
.end
