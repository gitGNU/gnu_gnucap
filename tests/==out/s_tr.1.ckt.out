'c check 
v1 ( 1 0 )  DC pulse iv= 0. pv= 1. delay= 0. rise= 0.01 fall= 0.01 width= 1.K period= 1.K  AC  1.
r1 ( 1 2 )  1.
c2 ( 2 0 )  1.
#Time       v(1)       v(2)      
 0.         0.         0.        
 0.001      0.099999   99.899u   
 0.002      0.2        249.72u   
 0.003      0.3        499.35u   
 0.004      0.4        848.67u   
 0.005      0.5        0.0012976 
 0.01       0.99999    0.0050317 
 0.014286   0.99999    0.0092867 
 0.018571   0.99999    0.013524  
 0.022857   0.99999    0.017742  
 0.031429   0.99999    0.026126  
 0.1        0.99999    0.090691  
 0.2        0.99999    0.17729   
 0.3        0.99999    0.25564   
 0.4        0.99999    0.32653   
 0.5        0.99999    0.39067   
 0.6        0.99999    0.4487    
 0.7        1.         0.50121   
 0.8        1.         0.54871   
 0.9        1.         0.59169   
 1.         1.         0.63058   
#Time       v(1)       v(2)      
 1.         1.         0.63058   
 1.001      1.         0.63095   
 1.1        1.         0.66576   
 1.2        1.         0.69759   
 1.3        1.         0.72639   
 1.4        1.         0.75245   
 1.5        1.         0.77603   
 1.6        1.         0.79736   
 1.7        1.         0.81666   
 1.8        1.         0.83412   
 1.9        1.         0.84991   
 2.         1.         0.86421   
#Time       v(1)       v(2)      
 4.         1.         0.98165   
 4.1        1.         0.9834    
 4.2        1.         0.98498   
 4.3        1.         0.98641   
 4.4        1.         0.98771   
 4.5        1.         0.98888   
 4.6        1.         0.98994   
 4.7        1.         0.99089   
 4.8        1.         0.99176   
 4.9        1.         0.99255   
 5.         1.         0.99326   
#Time       v(1)       v(2)      
 5.         1.         0.99326   
 5.001      1.         0.99326   
 5.1        1.         0.9939    
 5.2        1.         0.99448   
 5.3        1.         0.99501   
 5.4        1.         0.99548   
 5.5        1.         0.99591   
 5.6        1.         0.9963    
 5.7        1.         0.99665   
 5.8        1.         0.99697   
 5.9        1.         0.99726   
 6.         1.         0.99752   
 6.1        1.         0.99776   
 6.2        1.         0.99797   
 6.3        1.         0.99816   
 6.4        1.         0.99834   
 6.5        1.         0.9985    
 6.6        1.         0.99864   
 6.7        1.         0.99877   
 6.8        1.         0.99889   
 6.9        1.         0.99899   
 7.         1.         0.99909   
 7.1        1.         0.99918   
 7.2        1.         0.99925   
 7.3        1.         0.99933   
 7.4        1.         0.99939   
 7.5        1.         0.99945   
 7.6        1.         0.9995    
 7.7        1.         0.99955   
 7.8        1.         0.99959   
 7.9        1.         0.99963   
 8.         1.         0.99967   
 8.1        1.         0.9997    
 8.2        1.         0.99973   
 8.3        1.         0.99975   
 8.4        1.         0.99978   
 8.5        1.         0.9998    
 8.6        1.         0.99982   
 8.7        1.         0.99983   
 8.8        1.         0.99985   
 8.9        1.         0.99986   
 9.         1.         0.99988   
#Time       v(1)       v(2)      
 9.         1.         0.99988   
 9.001      1.         0.99988   
 9.1        1.         0.99989   
 9.2        1.         0.9999    
 9.3        1.         0.99991   
 9.4        1.         0.99992   
 9.5        1.         0.99993   
 9.6        1.         0.99993   
 9.7        1.         0.99994   
 9.8        1.         0.99994   
 9.9        1.         0.99995   
 10.        1.         0.99995   
Gnucap   System status
iterations: op=0, dc=0, tran=34, fourier=0, total=471
transient timesteps: accepted=119, rejected=1, total=120
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%
